api/views.py

```diff
@@ -158,6 +169,15 @@ def add_playlist_detail(self, qs):
         """
         return qs.select_related('channel')
 
+    def add_billing_account_item_detail(self, qs):
```

Comment:
In the spirit of YAGNI, my preference (not strong) would be to add this when you need it.